Graphs/TreeGraph/index.js

```diff
@@ -7,15 +7,30 @@ import AbstractGraph from "../AbstractGraph";
 import { properties } from "./default.config";
 import { List } from 'immutable';
 import './styles.css'
+import { parseSrc } from '@/lib/ui-components/utils.js'
 
-const diagonal = (s, d) => {
+const rectNode = { width : 120, height : 45, textMargin : 5 }
```

Comment:
will this change the height automatically or do we need to keep the width/height constant? I am ok with either one!